versioned_sidebars/version-3.9-sidebars.json

```
@@ -197,51 +437,682 @@
+              "label": "Helm Chart を使用して ScalarDL Auditor をデプロイ"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/how-to-deploy-scalar-admin-for-kubernetes",
+              "label": "Kubernetes 用の Scalar Admin をデプロイ"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/mount-files-or-volumes-on-scalar-pods",
+              "label": "ScalarDL ポッドにファイルまたはボリュームをマウント"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/use-secret-for-credentials",
+              "label": "シークレットリソースを使用"
+            }
+          ]
+        },
+        {
+          "type": "doc",
+          "id": "scalar-licensing/README",
+          "label": "製品ライセンスキーの設定"
+        },
+        {
+          "type": "doc",
+          "id": "installation-with-docker",
+          "label": "ローカル環境に ScalarDL をインストール"
+        },
+        {
+          "type": "doc",
+          "id": "ca/caserver-getting-started",
+          "label": "CFSSL で CA サーバーを起動"
+        }
+      ]
+    },
+    {
+      "type": "category",
+      "label": "運用",
+      "collapsible": true,
+      "items": [
+        {
+          "type": "category",
+          "label": "Scalar Kubernetes",
+          "collapsible": true,
+          "items": [
+            {
+              "type": "category",
+              "label": "モニタリングガイド",
+              "collapsible": true,
+              "items": [
+                {
+                  "type": "category",
+                  "label": "アラート",
+                  "collapsible": true,
+                  "items": [
+                    {
+                      "type": "doc",
+                      "id": "scalar-kubernetes/alerts/Envoy",
+                      "label": "Envoy アラート"
+                    },
+                    {
+                      "type": "doc",
+                      "id": "scalar-kubernetes/alerts/Ledger",
+                      "label": "Ledger アラート"
+                    }
+                  ]
+                },
+                {
+                  "type": "doc",
+                  "id": "scalar-kubernetes/K8sMonitorGuide",
+                  "label": "Kubernetes モニタリングガイド"
+                },
+                {
+                  "type": "doc",
+                  "id": "scalar-kubernetes/K8sLogCollectionGuide",
+                  "label": "Kubernetes ログ収集ガイド"
+                }
+              ]
+            },
+            {
+              "type": "category",
+              "label": "バックアップと復元のガイド",
+              "collapsible": true,
+              "items": [
+                {
+                  "type": "doc",
+                  "id": "scalar-kubernetes/BackupRestoreGuide",
+                  "label": "Kubernetes 環境で ScalarDL データをバックアップおよび復元"
+                },
+                {
+                  "type": "doc",
+                  "id": "scalar-kubernetes/BackupRDB",
+                  "label": "Kubernetes で RDB をバックアップ"
```

```suggestion
                  "label": "Kubernetes 環境で RDB をバックアップ"
```
Same as `sidebars.js`.